docs/api/implementation-guide.md

```diff
@@ -0,0 +1,53 @@
+
+Finally, the [package metadata resource](./registration-base-url-resource.md) and [search resource](./search-query-service-resource.md) must contain the hosted URL in the `iconUrl`, `licenseUrl`, and/or `readmeUrl` properties of the JSON response.
+Packages (`.nupkg` files) are immutable and must not be modified.
+
+Note that the license could be an SPDX expression, or an embedded file (but not both).
+Packages that use a license expression, when represented in search and package metadata results, can have the `licenseUrl` set to the license expression, URL encoded, and appended to the end of https://licenses.nuget.org/.
+For example, https://licenses.nuget.org/Apache-2.0.
+
+## Owner field
+
+The [package manifest file (`.nuspec`)](../reference/nuspec.md) have two fields, `<authors>` and `<owners>`.
+Package authors who are packaging third-party content often put the third-party name in the `<authors>` field.
+The `<owners>` field is intended to denote who published the package on a repository, and therefore who should be contacted in case of packing issues or questions.
+
+The `<owners>` field has been deprecated from the `.nuspec` file, so if packages contain this metadata, it should be ignored.
+Do not return the value of the `.nuspec` file's `<owners>` field in the `owners` property in the search resource or package metadata resource JSON response.
+
+If your repository has per-package permissions, it is recommended to report the accounts that have permissions to publish new versions in the search and package metadata resources JSON responses.
+
+## Known vulnerability and deprecation data
+
+The [Package Metadata Resource](./registration-base-url-resource.md) can contain [deprecation](./registration-base-url-resource.md#package-deprecation) and [vulnerability](./registration-base-url-resource.md#vulnerabilities) information.
+This allows customers browsing packages in Visual Studio's Package Manager User Interface, or equivalent in other IDEs, to be notified of important security or maintenance issues.
+
+Additionally, in order to provide high-performance vulnerability scanning during package restore, NuGet also downloads the full list of known vulnerabilities from [the `VulnerabilityInfo` resource](./vulnerability-info.md).
+
+If your package repository is "up-sourcing" packages from another repository, in order to mirror packages in your own feed, we recommend periodically checking the original source if there is deprecation or vulnerability data, and mirror that metadata in your own repository.
+If your package repository is up-sourcing from nuget.org specifically, by keeping state of the last time you checked (a "cursor"), you can use [the `Catalog` resource](./catalog-resource.md) to efficiently check if there are any package updates for packages you're mirroring, without having to download a large number of package metadata JSON files from the upstream feed.
```

Comment:
@joelverhagen is this right?  Does nuget.org create a new catalog leaf when deprecation and/or vulnerability info changes?

Reply:
Yes, it does. You can consider linking to this how-to in addition to the catalog resource: https://learn.microsoft.com/en-us/nuget/guides/api/query-for-all-published-packages